src/server/db_slice.cc

```diff
@@ -598,60 +602,54 @@ void DbSlice::FlushSlots(SlotSet slot_ids) {
   }).Detach();
 }
 
-void DbSlice::FlushDb(DbIndex db_ind) {
+void DbSlice::FlushDbIndexes(const std::vector<DbIndex>& indexes) {
```

Comment:
This is refactoring the FlushDb and FlushAll to have the same logic, 
before this change FlushAll was missing the PerformDeletion calls with creating new fiber
I also added here a call to tiered->CancelAllIos which will cancel all pending/enqueued io disk writes

Reply:
I will add this test